src/utils/loadScript.ts

```diff
@@ -0,0 +1,17 @@
+import { HTMLAttributes } from "react";
+
+export function loadScript(
+  src: string,
+  onLoad: () => void,
```

Comment:
nit: considering it's going into a property called `onload`, is this better off being called `onload` instead of `onLoad`? Open to opinions here.